.php_cs.dist

```diff
@@ -34,7 +34,8 @@ return PhpCsFixer\Config::create()
     ])
     ->setFinder(
         PhpCsFixer\Finder::create()
-            ->in(__DIR__ . '/tests/PhpCoveralls')
+            ->in(__DIR__ . '/tests')
             ->in(__DIR__ . '/src')
+            ->exclude('Fixture')
```

Comment:
Doesn't work right with `tests/Fixture`. What shall I do?

Reply:
change dir/src and dir/tests to __dir__